### PR TITLE
configure a folder and a group for a shared git storage

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: puppet/modules/secretsgit/manifests/init.pp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+  file { $path:
+    ensure => directory,
+    owner => 'root',
+    group => $group,
+    mode => '2770',
+  }
+
+  $users.each |String $user| {
```
Comment (ekohl) [inline puppet/modules/secretsgit/manifests/init.pp]: I think we can also use `members => $users` on `group`.

File: puppet/modules/secretsgit/manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  group { $group:
+    ensure => present
+  }
+
+  file { $path:
+    ensure => directory,
+    owner => 'root',
+    group => $group,
+    mode => '2770',
```
Comment (ekohl) [inline puppet/modules/secretsgit/manifests/init.pp]: Can you align the parameters?

File: puppet/modules/secretsgit/manifests/init.pp [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class secretsgit(
+  String $group = 'secretsgit',
+  Stdlib::Absolutepath $path = '/srv/secretsgit',
+  Array[String] $users = []
+) {
+
+  group { $group:
+    ensure => present
```
Comment (ekohl) [inline puppet/modules/secretsgit/manifests/init.pp]: I like trailing commas :)
```suggestion
    ensure => present,
```

File: puppet/modules/secretsgit/manifests/init.pp (hunk 2 above)
Comment (evgeni) [reply puppet/modules/secretsgit/manifests/init.pp]: I thought that's not a thing anymore in Puppet land? But I sure can.

File: puppet/modules/secretsgit/manifests/init.pp (hunk 1 above)
Comment (evgeni) [reply puppet/modules/secretsgit/manifests/init.pp]: the [docs](https://puppet.com/docs/puppet/6.6/types/group.html#group-attribute-members) say "Requires features manages_members", and [`groupadd`](https://puppet.com/docs/puppet/6.6/types/group.html#group-provider-groupadd), the default implementation Puppet uses on Linux [does not have that feature](https://puppet.com/docs/puppet/6.6/types/group.html#group-provider-features).

**Pull request merged**